tracer/src/Datadog.Trace/Debugger/Instrumentation/MethodDebuggerInvoker.MultiProbe.cs

```diff
@@ -0,0 +1,197 @@
+// <copyright file="MethodDebuggerInvoker.MultiProbe.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using Datadog.Trace.Debugger.Configurations.Models;
+using Datadog.Trace.Debugger.Expressions;
+using Datadog.Trace.Debugger.Instrumentation.Collections;
+using Datadog.Trace.Debugger.RateLimiting;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Debugger.Instrumentation
+{
+    /// <summary>
+    /// MethodDebuggerInvoker for multiple probes scenario (where there are more than one _method_ probe).
```

Comment:
```suggestion
    /// MethodDebuggerInvoker for multiple probes scenario (where there is more than one _method_ probe).
```

Reply:
Done in e3443b9b41459db08459c5cbadf67dd5b0bff45c.